Elements/src/Geometry/Line.cs

```diff
@@ -1045,6 +1045,18 @@ public bool TryGetOverlap(Line line, out Line overlap)
             return true;
         }
 
+        /// <summary>
+        /// Projects current line onto a plane
+        /// </summary>
+        /// <param name="plane">Plane to project</param>
+        /// <returns>New line on a plane</returns>
+        public Line Project(Plane plane)
```

Comment:
Can we call this `Projected`? Similar to my comment in #810, we use past tense in Elements for methods that return a new object.

Reply:
Thank you for suggestion. It was changed in [this commit.](https://github.com/hypar-io/Elements/pull/811/commits/1d9317e07549eb627ffb98912f27e6cadc2e7da5)